app/services/pieces_justificatives_service.rb

```diff
@@ -138,17 +138,26 @@ def self.pjs_for_commentaires(dossier)
       .map(&:piece_jointe)
   end
 
-  def self.pjs_for_dossier(dossier)
-    bill_signatures = dossier.dossier_operation_logs.filter_map(&:bill_signature).uniq
-
-    [
-      dossier.justificatif_motivation,
-      dossier.attestation&.pdf,
-      dossier.etablissement&.entreprise_attestation_sociale,
-      dossier.etablissement&.entreprise_attestation_fiscale,
-      dossier.dossier_operation_logs.map(&:serialized),
-      bill_signatures.map(&:serialized),
-      bill_signatures.map(&:signature)
-    ].flatten.compact
+  def self.pjs_for_dossier(dossier, claimant_expert = false)
+    if claimant_expert
+      [
+        dossier.justificatif_motivation,
+        dossier.attestation&.pdf,
+        dossier.etablissement&.entreprise_attestation_sociale,
+        dossier.etablissement&.entreprise_attestation_fiscale
+      ].flatten.compact
+    else
+      bill_signatures = dossier.dossier_operation_logs.filter_map(&:bill_signature).uniq
+
+      [
+        dossier.justificatif_motivation,
+        dossier.attestation&.pdf,
+        dossier.etablissement&.entreprise_attestation_sociale,
+        dossier.etablissement&.entreprise_attestation_fiscale,
+        dossier.dossier_operation_logs.map(&:serialized),
+        bill_signatures.map(&:serialized),
+        bill_signatures.map(&:signature)
+      ].flatten.compact
+    end
```

Comment:
Pour éviter de répéter du code, on devrait pouvoir faire un truc du genre :

```suggestion
    pjs = [
        dossier.justificatif_motivation,
        dossier.attestation&.pdf,
        dossier.etablissement&.entreprise_attestation_sociale,
        dossier.etablissement&.entreprise_attestation_fiscale
      ].flatten.compact
    
    if !for_expert
      bill_signatures = dossier.dossier_operation_logs.filter_map(&:bill_signature).uniq

      pjs += [
        dossier.dossier_operation_logs.map(&:serialized),
        bill_signatures.map(&:serialized),
        bill_signatures.map(&:signature)
      ].flatten.compact
    end
```